docs/configuration.rst

```diff
@@ -986,6 +986,134 @@ remain available as described at the start of the :ref:`Configuration` section.
     Name of the :term:`Provider` used for login. This represents the identifier that is set to define how to
     differentiate between a local sign-in procedure and a dispatched one some known :ref:`authn_providers`.
 
+Network Mode Settings
+~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration parameters are related to Magpie's "Network Mode" which allows networked instances of Magpie
```

Comment:
``` :ref:`network_mode` ``` here and below

``` `Magpie` ``` here and below. Also consider replacing by more specific terms when possible, such as reusing the `Network Node` term instead of `Magpie instance`.